src/main/java/com/patchworkmc/manifest/converter/GloomDefenitionParser.java

```diff
@@ -0,0 +1,48 @@
+package com.patchworkmc.manifest.converter;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+import io.github.fukkitmc.gloom.definitions.ClassDefinition;
+import io.github.fukkitmc.gloom.definitions.GloomDefinitions;
+import io.github.fukkitmc.gloom.definitions.SelfMember;
+
+import com.patchworkmc.manifest.accesstransformer.AccessTransformerEntry;
+import com.patchworkmc.manifest.accesstransformer.AccessTransformerList;
+
+public class GloomDefenitionParser {
+	private GloomDefenitionParser() {
+		// NO-OP
+	}
+
+	public static GloomDefinitions parse(AccessTransformerList list) {
+		HashMap<String, MemberHolder> definitionMap = new HashMap<>();
+
+		for (AccessTransformerEntry entry : list.getEntries()) {
+			MemberHolder classDefinition = new MemberHolder();
+			definitionMap.put(entry.getClassName(), classDefinition);
```

Comment:
This will overwrite the existing `MemberHolder` with an empty one each time. Instead. you should use `computeIfAbsent`.